tools/mason/MasonSearch.chpl

```diff
@@ -65,7 +65,13 @@ proc masonSearch(origArgs : [] string) {
           }
         }  else {
           const ver = findLatest(searchDir + dir);
-          results.push_back(name + " (" + ver.str() + ")");
+          if ver[1] == false {
```

Comment:
With the change suggested below and dropping the warning here, we should be able to do something like:

```chpl
// In MasonUtil.chpl
const versionZero = new VersionInfo(0, 0, 0);
```

```chpl
if ver != versionZero then
  results.push_back(name + " (" + ver.str() + ")");
}
```